packages/react/cypress/component/auto/form/ShadcnAutoBelongsToInput.cy.tsx

```diff
@@ -0,0 +1,212 @@
+import React from "react";
+import { elements } from "../../../../spec/auto/shadcn-defaults/index.js";
+import { apiTriggerOnly } from "../../../../spec/auto/support/Triggers.js";
+import { makeAutocomponents } from "../../../../src/auto/shadcn/index.js";
+import { api } from "../../../support/api.js";
+import { ShadcnWrapper } from "../../../support/auto.js";
+
+const { AutoForm, AutoSubmit, SubmitResultBanner, AutoBelongsToInput } = makeAutocomponents(elements);
+
+describe("ShadcnAutoBelongsToInput", () => {
```

Comment:
I think Millan recently made a version of this test that worked against both design libs